firmware/OBC/teensy/teensy.ino

```diff
@@ -267,5 +277,29 @@ void loop()
     }
 #endif
 
+#ifdef USING_LORA
+  if (transmittedFlag) {
+    transmittedFlag = false;
+    
+  }
+  if (receivedFlag) {
+    receivedFlag = false;
+
+  }
+  if (operationDone) {
+    operationDone = false;
+
+  }
+  if (scanDone) {
+    scanDone = false;
+
+  }
+#endif
+
+  if (lora.GetMode() != LORA_MODE::TRANSMIT) {
+    lora.InterruptTransmit(gps_msg.latitude)
+    lora.InterruptTransmit(gps_msg.longitude)
+    lora.InterruptTransmit(gps_msg.altitude) 

```

Comment:
`latitude`, `longitude`, and `altitude` are all of type double. To transmit these values, we'll need to convert these into byte arrays. The best way is using unions:
```c
typedef union LORA_MSG
{
    byte arr[8];
    double val;
} LORA_MSG;
```
Example of use:
```c
LORA_MSG msg;
msg.val = gps_msg.latitude;
lora.InterruptTransmit(msg.arr);
```